UTNAK/kepler16b-using-imce-vocabulary
# 11 update oml descriptions based on simulations run on model

## Checklist before submitting a merge request


## Description of contribution

In #6 , work has been done to simulate OML descriptions. 
The PR adds the capability of updating these OML descriptions using the SPARQL Update Query.

Close #11 


## Testing performed


## How to Test Expected functionality changes

- [ ] I have added the steps needed to test these changes

1. Go to https://github.com/UTNAK/kepler16b-using-imce-vocabulary
2. Switch to `11-update-oml-descriptions-based-on-simulations-run-on-model` branch
3. Run `git pull`
4. From the command line run `./gradlew load` or from the `Setup Tasks` webview
5. Wait for that command to finish
6. Open the `src/analysis/analysisShell.qmd`
7. Run Cell by Cell
8. From the command line run `./gradlew save` or from the `Save Tasks` webview
9. Verify that the `src/oml/example.com/tutorial2/description/statedictionary.oml` is updated




## Additional context

[If needed, you may add additional context]
